### PR TITLE
tiny-js-world

### PR DESCRIPTION
**Pull request by Yasya23**
### tiny-js-world

[Demo](https://yasya23.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/Yasya23/a-tiny-JS-world/tree/gh-pages)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Yasya23** [comment]
I checked for common errors, I hope everything is ok

File: submissions/Yasya23/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+};
+
+const catWoman = { ...cat };
+catWoman.name = "Luci";
+catWoman.gender = "female";
+
+const inhabitants = [dog, cat, man, woman, catWoman];
+const properties = ["species", "name", "gender", "legs", "hands", "saying"];
+
+inhabitants.forEach(function (elem) {
```
Comment (OleksiyRudenko) [inline submissions/Yasya23/tiny-js-world/index.js:53]: Here we can safely use an arrow function. Less letters - easier to read.

`elem` is a generic term.
From the checklist:
> Give variables and functions meaningful names. Avoid generic names like `item`, `element`, `key`, `object`, `array` or their variations. Exception: helper functions that are specifically and intentionally designed to be multipurpose.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@Yasya23 well done!
Let's polish it.

File: submissions/Yasya23/tiny-js-world/index.js (hunk 1 above)
Comment (Yasya23) [reply submissions/Yasya23/tiny-js-world/index.js]: Thanks for checking! Fixed it.

File: submissions/Yasya23/tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  let result = properties.map((key) => elem[key]);
+  print(result.join("; "));
```
[the remark this answers is not in the document]
Comment (Yasya23) [reply submissions/Yasya23/tiny-js-world/index.js]: Fixed it.

**Review by OleksiyRudenko — APPROVED** [review]
@Yasya23 well done!

**Pull request merged**